x/attribute/types/events.go

```diff
@@ -1,20 +1,28 @@
 package types
 
-import "encoding/base64"
+import (
+	"encoding/base64"
+	time "time"
+)
 
 const (
-	// The type of event generated when account attributes are added.
+	// EventTypeAttributeAdded emitted when account attributes are added.
 	EventTypeAttributeAdded string = "account_attribute_added"
-	// The type of event generated when account attributes are updated.
+	// EventTypeAttributeUpdated emitted when account attributes are updated.
 	EventTypeAttributeUpdated string = "account_attribute_updated"
-	// The type of event generated when account attributes are removed.
+	// EventTypeAttributeUpdated emitted when a attribute's expiration date is updated.
+	EventTypeAttributeExpirationUpdated string = "account_attribute_expiration_updated"
+	// EventTypeAttributeDeleted emitted when account attributes are removed.
 	EventTypeAttributeDeleted string = "account_attribute_deleted"
-	// The type of event generated when a distinct account attribute is deleted.
+	// EventTypeAttributeDistinctDeleted emitted when a distinct account attribute is deleted.
 	EventTypeAttributeDistinctDeleted string = "account_attribute_distinct_deleted"
+	// EventTypeDeletedExpired emitted when attributes have expired and been deleted in begin blocker
+	EventTypeDeletedExpired string = "attribute_deleted_expired"
 
-	AttributeKeyAttribute      string = "attribute"
-	AttributeKeyNameAttribute  string = "attribute_name"
-	AttributeKeyAccountAddress string = "account_address"
+	AttributeKeyAttribute           string = "attribute"
+	AttributeKeyNameAttribute       string = "attribute_name"
+	AttributeKeyAccountAddress      string = "account_address"
+	AttributeKeyTotalExpiredDeleted string = "total_expired_deleted"
```

Comment:
`AttributeKeyTotalExpiredDeleted` could be `AttributeKeyTotalExpired`

Reply:
Fixed.